news/4377.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Update hooks related to PySide2.QtWebEngineWidgets, ensure the relevant
+supporting files required for a QtWebEngineView are copied into the 
+distribution.
```

Comment:
Thanks for the changle entry. Please rename it into `…hooks…`, since all hook related changes go there.

Reply:
Renamed